app/lib/actions/pulp/repository_group/create.rb

```diff
@@ -0,0 +1,27 @@
+module Actions
+  module Pulp
+    module RepositoryGroup
+      class Create < Pulp::Abstract
+        include Helpers::Presenter
+
+        input_format do
+          param :id
+          param :pulp_ids
+        end
+
+        def run
+          pulp_resources.
+            repository_group.create(input[:id],
+                                    :id => input[:id],
+                                    :repo_ids => input[:pulp_ids],
+                                    :display_name => "temporary group for export",
+                                    :distributors => [group_export_distributor])
+        end
```

Comment:
ya, it's needed because Pulp needs to know all of the repos you're exporting in advance, so it can cut one ISO instead of one ISO per repo.